specificatie/genereervariant/openapi.yaml

```diff
@@ -2187,7 +2211,12 @@ components:
         _links:
           $ref: '#/components/schemas/HalPaginationLinksMetLast'
         _embedded:
-          $ref: '#/components/schemas/ZoekResultaatHalCollectie__embedded'
+          type: object
```

Comment:
@strijm, toch een opmerking ondanks dat ik heb ge-approved. Hier wordt een object inline gedefinieerd. Dit gebeurt ook voor o.a. AdresHalCollectie__embedded en AdresseerbaarobjectHalCollectie__embedded. Dit zorgt ervoor dat je nu gegenereerde classes krijgt met namen _embedded, _embedded2, _embedded3 etc.
Kan je bij het genereren van de openapi.yaml ervoor zorgen dat het inline embedden niet gebeurt? Dit lijkt nl niet altijd te gebeuren. Bij bij AdresHal wordt AdresHal_embedded niet inline ge-embed. Mogelijk kan je hiermee achterhalen waarom het inline embedden plaatsvindt

Reply:
@MelvLee dat zit hem niet in de gegenereerde yaml maar in de originele yaml. Dat kan wel anders, maar dan moet ik de originele yaml aanpassen. Op zich geen probleem, maar ik denk dat er impact op de code is en aangezien we op het punt staan live te gaan met v1.1.0, lijkt me dit geen goed moment om dat nog als onderdeel van deze release te veranderen, tenzij dit een blocking issue is natuurlijk maar dan betekent dit, dat de release waarschijnlijk ook weer later wordt.
Zou jij een issue willen maken, zodat we dit in een volgende sprint mee kunnen nemen?

Reply:
@strijm , ik zie dat het inderdaad in de originele yaml zit. Het is m.i. niet blocking aangezien het alleen impact heeft op de code generatie. Ik zal het even met de anderen het erover hebben. Ik zal dan eventueel een issue aanmaken.